lib/core/src/tag/tag.tsx

```diff
@@ -8,17 +8,17 @@ export interface TagType {
 
 interface Props {
 	/**
-	 * The information that tag holds.
+	 * Label of the tag
 	 */
 	children: string;
 	/**
-	 * The color and background color of tag.
+	 * Color of the tag (import from Tag.colors)
```

Comment:
Don't use the word "import" as it may be confusing with the ES Modules import (when we import other libraries). Should change it to just "use one from Tag.colors"